test/fetch-groups-gh.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Tests for the fetch-groups module that require a W3C API key
+ *
+ * These tests are separated from the tests that do not require a W3C API key
```

Comment:
```suggestion
 * Tests for the fetch-groups module that require a GitHub token
 *
 * These tests are separated from the tests that do not require a GitHub token
```

... but in practice, do we still need to keep these tests separated? They had to because of the W3C API key secret. However, if I read the documentation correctly, the GitHub token should be set on pull requests, even on pull requests that originate from forked repositories:

> For workflows that are triggered by the pull_request_target event, the GITHUB_TOKEN is granted read/write repository permission unless the permissions key is specified and the workflow can access secrets, even when it is triggered from a fork
> https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target